.github/workflows/security.yml

```
@@ -4,211 +4,46 @@ on:
-          
-          if [ "${{ needs.secrets-scan.result }}" != "success" ] || [ "${{ needs.dependency-scan.result }}" != "success" ] || [ "${{ needs.static-analysis.result }}" != "success" ]; then
-            echo "⚠️ Security issues detected. Please review the reports." >> $GITHUB_STEP_SUMMARY
-            exit 1
-          else
-            echo "✅ All security scans passed!" >> $GITHUB_STEP_SUMMARY
-          fi
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Set up Python
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)